docs/fiddles/features/macos-dock-menu/main.js

```diff
@@ -1,21 +1,23 @@
-const { app, BrowserWindow, Menu } = require('electron')
+const { app, BrowserWindow, Menu } = require('electron');
+const path = require('path');
+const os = require('os');
 
 function createWindow () {
   const win = new BrowserWindow({
     width: 800,
     height: 600,
     webPreferences: {
-      nodeIntegration: true
+      preload: path.join(__dirname, 'preload.js')
     }
-  })
+  });
```

Comment:
Let's avoid the `;` changes for now, I think.

Reply:
Ok.